test/addons-napi/test_general/test_general.c

```diff
@@ -215,12 +216,35 @@ napi_value testAdjustExternalMemory(napi_env env, napi_callback_info info) {
   return result;
 }
 
+napi_value testNapiRun(napi_env env, napi_callback_info info) {
+  size_t string_size;
+  char *string;
```

Comment:
Pointer star goes with the type.

Reply:
We’re inconsistent about that in this file, and I think this can pass as “idiomatic for C code”